### PR TITLE
Add artik530 and artik533 getting started guides

### PR DESCRIPTION
**Pull request by zwalchuk**
Change-type: patch

### CODE REVIEW

File: shared/whatYouNeed/artik530.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<img style="float: right;padding-left: 10px;" src="/img/{{ $device.id }}/{{ $device.id }}.jpg" width="25%">
+* A [{{ $device.name }}][device] module mounted on a development board.
+* A 5VDC power supply (100-240V input), if you purchased the development kit you should have one included.
+* At least one wireless communication antenna, also provided as part of the development kit.
```
Comment (lurch) [inline shared/whatYouNeed/artik530.md]: Is the antenna optional, if you're connecting via an Ethernet cable?

File: shared/whatYouNeed/artik530.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+* **[optional]** Ethernet cable, if you prefer to **not** connect over wifi.
+* A 4GB or larger SD card. The [speed class][sdSpeed] of the card also matters - class 10 card or above is the way to go.
+* **[optional]** A tooth pick or small screw driver to set the boot switches.
```
Comment (lurch) [inline shared/whatYouNeed/artik530.md]: I think "toothpick" and "screwdriver" should both be single words?

**Comment by lurch** [comment]
This PR adds `static/img/device/artik530.svg`, `static/img/device/artik533.svg` and `static/img/device/artik533s.svg` - presumably one of those is redundant?

**Comment by floion** [comment]
Right, as @lurch pointed out, `static/img/device/artik533.svg` should be removed @zwalchuk 
Thanks Andrew

File: shared/whatYouNeed/artik530.md (hunk 1 above)
Comment (floion) [reply shared/whatYouNeed/artik530.md]: Yeah, should be optional if wired is used

**Pull request merged**